listenbrainz/webserver/views/stats_api.py

```diff
@@ -550,3 +584,29 @@ def _get_entity_list(
     elif entity == 'recording':
         return getattr(stats, stats_range).recordings[offset:count]
     raise APIBadRequest("Unknown entity: %s" % entity)
+
+
+def _get_country_codes_by_msids(artist_msids):
+    country_map = defaultdict(int)
+    for index in range(0, len(artist_msids), 50):
```

Comment:
Once the dataset hoster allows POST, we should try to minimize the number of requests we need to send to it.